.github/ISSUE_TEMPLATE/b-bug-report.yml

```
@@ -0,0 +1,87 @@
+name: Report a bug
+description: You encountered a bug in the geolocator plugin that you would like to report.
+title: "Bug: "
+labels: ["needs-triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to submit an issue!
+        To effectively tackle it, we require as much information as possible.
```

Since we are asking a lot of information, lets try to really persuade users to be as complete as possible: 

```suggestion
        To effectively tackle it, we require as much information as possible. This might seem a lot of information but 
        will allow us to quickly help you and resolve the issue you are facing.
```